tests/unit/test_cli_analyze.py

```diff
@@ -11,13 +11,15 @@
 # TODO: These should be moved to the integration tests.
 
 
+@patch("databricks.labs.lakebridge.contexts.application.ApplicationContext.add_user_agent_extra", new=MagicMock())
```

Comment:
why do you need to patch given the config is already inside `workspace_client` and we already have a mock for it.

All we need is to add an additional assert may be to check the value for that key?